data/event-schemas/schema/m.room.create.yaml

```diff
@@ -6,7 +6,9 @@ properties:
   content:
     properties:
       creator:
-        description: The `user_id` of the room creator. This is set by the homeserver.
+        description: |-
```

Comment:
There are probably examples also -- should those use the format of the "latest" room version? Showing both sounds complicated.

(This question goes for all modified events, which I think is just `m.room.create` and `m.room.redaction`)

Reply:
best to remove it from the examples I think

Reply:
Updating the examples makes sense, should we also update the schemas or show both?

Reply:
I don't think I understand the question?

Reply:
There's both an event schema and an example event for each event type; do we want to update both?

Reply:
Never mind, I think we're *only* talking about the examples. Sorry.